### PR TITLE
Removes the Autodoc/Autosurgeon machine (Not the item)

### PR DESCRIPTION
**Pull request by francinum**
## About The Pull Request

Reverts #1175

Removes the autodoc.

## Why It's Good For The Game

The autodoc is a replacement for an interaction players should be having, One of the balancing components of some of the high-end implants is that it is literally impossible to insert them without getting someone else involved.

## Changelog
:cl:
del: The autodoc machine has been removed.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Good change.

**Comment by DatBoiTim** [comment]
While good for the time being, maybe eventually make the Autodoc require the abductor tools? So it's not available all the time but if someone either robusts an Ayy or finds the random loot, they can get a reward beyond advanced tools.

**Comment by francinum** [comment]
I see no reason for this item existing regardless of any possible costs. It completely sidesteps surgery in a majority of cases, even lategame I wouldn't consider it acceptable design.

**Comment by LivingBatteryBox** [comment]
Meh, relying on other people is how you get betrayed. Of course people want to sidestep it. Autodocs are particularly fitting for downstreams like NSV that have people scrambling to do a billion things to keep the ship flying without dying horribly.

**Comment by ghost** [comment]
> Meh, relying on other people is how you get betrayed. Of course people want to sidestep it. Autodocs are particularly fitting for downstreams like NSV that have people scrambling to do a billion things to keep the ship flying without dying horribly.

You do know that Franc is one of the NSV project leads?

**Comment by DatBoiTim** [comment]
> > Meh, relying on other people is how you get betrayed. Of course people want to sidestep it. Autodocs are particularly fitting for downstreams like NSV that have people scrambling to do a billion things to keep the ship flying without dying horribly.
> 
> 
> 
> You do know that Franc is one of the NSV project leads?

Credentials moment

**Comment by Effemu** [comment]
this is just going to remove any use from implants and cybernetics
nobody on sage voluntarily gets surgery unless its emergency like liver or brain trauma

also medbay gets bombed 50% of the rounds anyways, so no surgery for anyone

**Comment by francinum** [comment]
> this is just going to remove any use from implants and cybernetics
> nobody on sage voluntarily gets surgery unless its emergency like liver or brain trauma
> 
> also medbay gets bombed 50% of the rounds anyways, so no surgery for anyone

You're overstating things, but yes it will make implants less common.

That's a good thing.

**Comment by Archanial** [comment]
Keep it as admin only item, just remove it from being researched.

**Comment by Effemu** [comment]
make it cost 300 credits per use
and everyone is happy

**Comment by zeskorion** [comment]
Not being able to speedmerge this is my only regret about giving up the maintainer position

**Comment by ike709** [comment]
YOLO

**Comment by LivingBatteryBox** [comment]
Fuck you Ike. Put stuff like this to a vote.

**Comment by r1ks-iwnl** [comment]
He'd merge it with negative votes anyway lol

**Comment by DatBoiTim** [comment]
Madlad

**Comment by Effemu** [comment]
ike more like 
i merge bad prs

😔

**Comment by zeskorion** [comment]
Powergamers seething

**Comment by DatBoiTim** [comment]
> Powergamers seething

Implants now have a risk reward trade off, fewer unstoppable power gaming valid hunters with full augs and combat implants

**Comment by jlsnow301** [comment]
I can't fault this PR, medbay storage turns into a feeding frenzy once you build one

**Comment by Goblinu** [comment]
Honestly i was kinda annoyed over the fact that 50 mins into the game people were casually wooshing around with thrusters and jetpack and every implant imaginable, but i still think traitors should get tools to implant themselves (I would love to code myself a medical traitor item that autoimplants pills)

**Comment by Sinestia** [comment]
Bless you good sir

**Comment by Sinmero** [comment]
Welp no more implants on LRP

**Comment by byandaccount** [comment]
i don't play on bee regularly so i can't say how op this is but instead of axing it why not allow it to give you "exotic" upgrades that normal surgery wouldn't allow to begin with? on hippie the autodoc basically allows you to delimb any humanoid species including xenos and monkeys and allow people to use those funni limbs as augmentations.
While this sounds pointless it can lead to extremely funny moments like xenobio raising a xenomorph queen just to swap its head with a monkey or traitors hijacking peoples bodies via someone elses help.

TLDR give it a different interesting use, buffing or finding other uses for stuff is miles better than nerfing/removing content.

**Comment by ghost** [comment]
> i don't play on bee regularly so i can't say how op this is but instead of axing it why not allow it to give you "exotic" upgrades that normal surgery wouldn't allow to begin with? on hippie the autodoc basically allows you to delimb any humanoid species including xenos and monkeys and allow people to use those funni limbs as augmentations.
> While this sounds pointless it can lead to extremely funny moments like xenobio raising a xenomorph queen just to swap its head with a monkey or traitors hijacking peoples bodies via someone elses help.
> 
> TLDR give it a different interesting use, buffing or finding other uses for stuff is miles better than nerfing/removing content.

Half measures are always worse.

**Comment by ItsTisme** [comment]
Ngl it was pretty dumb assisnats would run in and combat augment themselves with 15 augments. The whole point was to get them installed by a docter
